app/Http/Requests/RequestForm.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Http\Requests\RequestForm;
+
+class RequestForm extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|max:255',
+        ];
+    }
+
+    public function messages()
+    {
+        return [
+            'required' => trans('message.required'),
+            'max' => trans('message.max'),
+        ];
+    }
+    public function attributes()
```

Comment:
function này e đặt ở đây để làm gì vậy?

Reply:
không cần làm thế nhưng vẫn giải quyết được vấn đề hiện tại của em.
Em nghiên cứu ở https://laravel.com/docs/5.5/validation#form-request-validation phần `Customizing The Error Messages` nhé